### PR TITLE
Add DVS Sample documentation

### PR DESCRIPTION
**Pull request by NenoStefanov**

### CODE REVIEW

File: DvsSample/README.md [hunk 1]
```diff
@@ -1 +1,145 @@
...
+```
+
+## DVS Signing
+
+For simplicity signing here is done for a simple string, but could be done for different resources like files or pictures, that could be represented as a string. Keep in mind, that better usage of DVS is to sign the document hash, not the whole file instead.
+
+<img src="images/sign_message_activity.png">
+
+The user needs to enter some text and click `SIGN` button. The DVS Identity signing PIN is required to sign the text.
+`CreateDocumentHashTask` is executed to create a transaction of type `DocumentDVSInfo` from the document text by making a `POST` request to the RPA backend service.
```
Comment (tatiNo5) [inline DvsSample/README.md]: CreateDocumentHashTask needs to be a link

Comment (tatiNo5) [reply DvsSample/README.md]: Is DocumentDVSInfo described in any docs to give a link to it?

File: DvsSample/README.md [hunk 2]
```diff
@@ -1 +1,145 @@
...
+}
+```
+
+## DVS Signing
+
+For simplicity signing here is done for a simple string, but could be done for different resources like files or pictures, that could be represented as a string. Keep in mind, that better usage of DVS is to sign the document hash, not the whole file instead.
+
+<img src="images/sign_message_activity.png">
+
+The user needs to enter some text and click `SIGN` button. The DVS Identity signing PIN is required to sign the text.
```
Comment (tatiNo5) [inline DvsSample/README.md]: The DVS Identity signing PIN -> The DVS Identity PIN... the signing word is part of DVS itself

File: DvsSample/README.md [hunk 3]
```diff
@@ -1 +1,145 @@
...
+`CreateDocumentHashTask` is executed to create a transaction of type `DocumentDVSInfo` from the document text by making a `POST` request to the RPA backend service.
+
+```
+new CreateDocumentHashTask(getString(R.string.access_code_service_base_url), document, new CreateDocumentHashTask.Callback() {
+  ...
+}).execute();
+```
+
+Once the RPA backend service has returned the `DocumentDVSInfo` to the mobile client app, [getMfaSdk().verifyDocumentHash](https://github.com/miracl/mfa-client-sdk-android#boolean-verifydocumenthashbyte-document-byte-documenthash) method verifies that the DocumentDVSInfo hash value is correct for the given document:
```
Comment (tatiNo5) [inline DvsSample/README.md]: ...method verifies that the DocumentDVSInfo hash value is correct for the given document -> method verifies that the transaction hash value is correct for the given document ... too much using of the DocumentDVSInfo word..

File: DvsSample/README.md [hunk 4]
```diff
@@ -1 +1,145 @@
...
+  ...
+}
+```
+
+When the returned hash value is verified, the user is asked for their signing identity PIN:
+
+<img src="images/enter_pin_for_signing.png">
+
+
+Then, to sign the transaction [getMfaSdk().sign](https://github.com/miracl/mfa-client-sdk-android#status-signuser-user-byte-documenthash-string-multifactor-int-epochtime-string-authztoken-signature-signature) method is used:
```
Comment (tatiNo5) [inline DvsSample/README.md]: coma after the transaction word

File: DvsSample/README.md [hunk 5]
```diff
@@ -1 +1,145 @@
...
+
+```
+SampleApplication.getMfaSdk().sign(SampleApplication.getLoggedUser(), documentDvsInfo.getHash().getBytes(), new String[]{pin}, documentDvsInfo.getTimestamp().intValue(), new MPinMfaAsync.Callback<Signature>() {
+  ...
+}
+
+```
+
+If the signing is successful, the returned signature is verified by `VerifySignatureTask`. The result is displayed to the screen:
```
Comment (tatiNo5) [inline DvsSample/README.md]: VerifySignatureTask needs to be a link

File: DvsSample/README.md (hunk 4 above)
Comment (todinov) [reply DvsSample/README.md]: comma*

**Comment by Foriger** [comment]
As we spoke: Add screenshot for `DVS REGISTER` button, and mention, those concrete Activities where DVS Registration and DVS Signing happen.

**Pull request merged**